mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/internal/FrontendLoggerRoutes.kt

```diff
@@ -10,7 +10,7 @@ import org.slf4j.MDC
 fun Route.frontendLoggerRoutes() {
     val logger = application.environment.log
 
-    route("/api/v1/logger/") {
+    route("/internal/logger/") {
```

Comment:
Har skrudd av `CallLogging` på routes som begynner med `/internal` (for å unngå å logge liveness/rediness), som vil føre til litt uheldige konsekvenser for dette endepunktet nå.
Enten må det legges under en annen route, eller så må vi endre på `CallLogging` under `Monitoring` i `:common:ktor`.

Reply:
Eller, nå tenkte jeg feil. Det vil jo fortsatt logges, bare ikke `CallLogging` for endepunktet. Det er kanskje ok? :thinking:

Reply:
Ah, ok. Skal fikse